decidim-core/app/services/decidim/send_push_notification.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require "webpush"
+
+module Decidim
+  # This class generates a notification based on the given event, for the given
+  # resource/recipient couple. It is intended to be used by the
+  # `Decidim::NotificationGenerator` class, which schedules a job for each recipient
+  # of the event, so that we can easily control which jobs fail.
+
+  class SendPushNotification
+    include ActionView::Helpers::UrlHelper
+
+    # Send the push notification. Returns `nil` if the user didn't allowed push notifications
+    # or if the subscription to push notifications doesn't exist
+    #
+    # Returns the result of the dispatch or nil if user or subscription are empty
+    def perform(notification)
+      return unless Rails.application.secrets.vapid[:enabled]
+
+      notification.user.notifications_subscriptions.values.map do |subscription|
+        message_params = notification_params(Decidim::PushNotificationPresenter.new(notification))
+        payload = build_payload(message_params, subscription)
+        # Capture webpush exceptions in order to avoid this call to be repeated by the background job runner
+        # Webpush::Error class is the parent class of all defined errors
+        begin
+          Webpush.payload_send(payload)
```

Comment:
I believe `Webpush.payload_send` is expecting keyword arguments:
https://github.com/zaru/webpush/blob/061ff9a88efebe2403a2a1c0dd0b66b96483ae9a/lib/webpush.rb#L38

So we should add double star here:
```suggestion
          Webpush.payload_send(**payload)
```

Otherwise I am getting an error from this line.